cmd/chart-repo/chart_repo.go

```diff
@@ -24,7 +24,7 @@ import (
 
 var rootCmd = &cobra.Command{
 	Use:   "chart-repo",
-	Short: "Kubeapps Chart Repository utility",
+	Short: "Monocular Chart Repository utility",
```

Comment:
Perhaps we can just call this "Chart Repository utility" since it is a standalone tool used in multiple projects.